src/handbook/peopleops/code-of-conduct.md

```diff
@@ -0,0 +1,121 @@
+matters -- online and in-person -- as well as in all one-on-one communications
+with customers and staff pertaining to FlowForge business.
+
+This Code of Conduct also applies to unacceptable behavior occurring outside the
+scope of business activities when such behavior has the potential to adversely
+affect the safety and well-being of FlowForge staff and clients.
+
+### Culture and Citizenship
+
+A supplemental goal of this Code of Conduct is to increase open citizenship by
+encouraging participants to recognize the relationships between our actions and
+their effects within FlowForge culture.
+
+At the core are our [company values](../company/values.md). This Code of Conduct
+makes explict how we should behave whilst uphold our values.
+
+ - **Be inclusive**. We strive to be a company that welcomes and supports people
+   of all backgrounds and identities. This includes, but is not limited to members
+   of any race, ethnicity, culture, national origin, color, immigration status,
+   social and economic class, educational level, sexual orientation, gender
+   identity and expression, age, size, family status, political belief, religion,
+   and mental and physical ability.
+ - **Be considerate**. Your work at FlowForge will be used by other people, and
+   you in turn will depend on the work of others. Any decision you take will
+   affect users and colleagues, and you should take those consequences into account
+   when making decisions.
+ - **Be respectful**. Not all of us will agree all the time, but disagreement is
+   no excuse for poor behavior and poor manners. We might all experience some
+   frustration now and then, but we cannot allow that frustration to turn into a
+   personal attack. It’s important to remember that a company where people feel
+   uncomfortable or threatened is neither productive nor pleasant. FlowForge staff
+   should always be respectful when dealing with other personnel as well as with
+   people outside of FlowForge employment.
+
+### Acceptable and Expected Behavior
+
+The following behaviors are expected and requested of all FlowForge staff:
+
+ - Participate in an authentic and active way. In doing so, you contribute to the
+   health and longevity of FlowForge.
+ - Exercise consideration and respect in your speech and actions at all times.
+ - Attempt collaboration before conflict.
+ - Refrain from demeaning, discriminatory, or harassing behavior and speech.
+ - Be mindful of your surroundings and of your fellow participants. Alert FlowForge
+   leaders if you notice a dangerous situation, someone in distress, or violations
+   of this Code of Conduct, even if they seem inconsequential.
+
+### Unacceptable Behavior
+
+The following behaviors are considered harassment and are unacceptable within our community:
+
+ - Violence, threats of violence or violent language directed against another person.
+ - Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
```

Comment:
I didn't know what this was.

```suggestion
 - Sexist, racist, homophobic, transphobic, [ableist](https://en.wikipedia.org/wiki/Ableism) or otherwise discriminatory jokes and language.
```